### DISCUSSION TITLE
10.9.7, TV resources shown no resources seasons,I DON'T need this.

### DISCUSSION THREAD

**Discussion by Travel2Here** [jellyfin/jellyfin#12177] [General] [answered]
In version 10.9.7, other seasons are also shown in the season category for resources that don't exist, which is not what is expected from resource management. Am I setting something up wrong or did this version just change the way this works?

**Comment by Travel2Here** [comment]
<img width="926" alt="image" src="https://github.com/jellyfin/jellyfin/assets/99887507/6df08488-4c9f-4cd2-a9ec-2cfc996a38f0">
like this.i dont have season 1 and 2.

**Comment by TheMelmacian** [comment]
I had the same problem. Trying to delete them resulted in an error message but I couldn't find anything specific in the logs.
The workaround to get rid of them is to create the season folder, trigger a rescan of the show and then delete the empty seasons from the GUI.

I deactiveted the "Missing Episode Fetcher" as metadata provider in my tv show library. Let's see if this solves the problem or if the problem resurfaces.

**Reply by Travel2Here** [reply]
😂 bad mood

**Comment by Travel2Here** [comment, marked as answer]
Resolved. **Finally, it's solved.** 
Just leave this option **unchecked.**

but i still think this is a bug.

<img width="1259" alt="image" src="https://github.com/jellyfin/jellyfin/assets/99887507/932b1e62-f08f-4592-869d-80ee2d7fa5cf">

**Reply by TheMelmacian** [reply]
That maybe removes them from the GUI but the empty seasons most probably still exist in the database.

And for someone that is using the "merge series" function like myself (different folders for DVD/BluRay backups and other sources), it can only be a temporary workaround to deactivate it.

I will activate the "missing episode fetcher" again and if the bug resurfaces I will create an issue (if no one else does it in the meantime).
